Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -915,6 +923,63 @@ private void chooseController()
     alertDialog.show();
   }
 
+  public static void handleWiiOverlayDisablesControllersSetting(int selection, boolean IsFirstRun)
+  {
+    if (NativeLibrary
+            .GetConfig(SettingsFile.FILE_NAME_DOLPHIN + ".ini", Settings.SECTION_INI_ANDROID,
+                    SettingsFile.KEY_WII_OVERLAY_DISABLES_CONTROLLERS, "False")
+            .equals("True"))
+    {
+      // TODO: Add real controllers to Wii Input overlay so they can be changed by this setting.
+      if (!NativeLibrary
+              .GetConfig(SettingsFile.FILE_NAME_DOLPHIN + ".ini", Settings.SECTION_INI_CORE,
+                      SettingsFile.KEY_GCPAD_PLAYER_1, InputOverlay.EMULATED_GAMECUBE_CONTROLLER)
+              .equals(InputOverlay.GAMECUBE_ADAPTER))
+      {
+        NativeLibrary
+                .SetConfig(SettingsFile.FILE_NAME_DOLPHIN + ".ini", Settings.SECTION_INI_CORE,
+                        SettingsFile.KEY_GCPAD_PLAYER_1,
+                        selection == InputOverlay.OVERLAY_GAMECUBE ?
+                                InputOverlay.EMULATED_GAMECUBE_CONTROLLER :
+                                InputOverlay.DISABLED_GAMECUBE_CONTROLLER);
+
+        NativeLibrary.SetOverlaySIDevice(selection == InputOverlay.OVERLAY_GAMECUBE ?
+                Integer.parseInt(InputOverlay.EMULATED_GAMECUBE_CONTROLLER) :
+                Integer.parseInt(InputOverlay.DISABLED_GAMECUBE_CONTROLLER));
+      }
+
+      if (!NativeLibrary
+              .GetConfig(SettingsFile.FILE_NAME_WIIMOTE + ".ini", SettingsFile.KEY_WIIMOTE_PLAYER_1,
+                      SettingsFile.KEY_WIIMOTE_TYPE, "1").equals("2"))
```

Comment:
Any particular reason why "1" and "2" (and equivalents in the code below) aren't using constants?

Reply:
There are multiple instances of these values needing constants in code unrelated to this PR. If we're assigning constants to these values, I'd rather do it in a separate PR where all of those cases are handled.